packages/camera/camera_android_camerax/pubspec.yaml

```diff
@@ -0,0 +1,73 @@
+name: camera_android_camerax
+description: A new Flutter plugin project.
+repository: https://github.com/flutter/plugins/tree/main/packages/camera/camera_android_camerax
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
+homepage:
+publish_to: 'none'
+
+environment:
+  sdk: '>=2.14.0 <3.0.0'
+  flutter: ">=2.5.0"
```

Comment:
3.0 (or at least nothing earlier than 2.8, which is the earliest we have CI for).